### PR TITLE
[Merged by Bors] - Oracle cleanup

### PR DESCRIPTION
**Pull request by noamnelke**
## Motivation
Linting errors in the oracle package.

## Changes
Fix linting errors.
Move files to more appropriate packages, mainly to allow unexporting objects only used in tests.

### CODE REVIEW

**Comment by noamnelke** [comment]
bors try

File: cmd/hare/oracle.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package main
```
Comment (antonlerner) [inline cmd/hare/oracle.go:1]: add package desc for godoc

File: cmd/hare/oracle.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+type hareOracle struct {
+	oc *oracleClient
+}
+
+func newHareOracleFromClient(oc *oracleClient) *hareOracle {
+	return &hareOracle{oc: oc}
+}
+
+func (bo *hareOracle) IsIdentityActiveOnConsensusView(string, types.LayerID) (bool, error) {
```
Comment (antonlerner) [inline cmd/hare/oracle.go:15]: don't need to comment on that?

**Comment by noamnelke** [comment]
bors try

File: cmd/hare/oracle.go (hunk 2 above)
Comment (noamnelke) [reply cmd/hare/oracle.go:15]: no, `hareOracle` is not exported, so golint ignores it.

File: cmd/hare/oracle.go (hunk 1 above)
Comment (noamnelke) [reply cmd/hare/oracle.go:1]: This particular package is not part of this PR, I just moved stuff there.

I will add it for the `oracle` package, though.

**Comment by noamnelke** [comment]
bors merge

**Review by antonlerner — APPROVED** [review]

**Comment by noamnelke** [comment]
bors merge

**Comment by noamnelke** [comment]
bors merge

**Comment by noamnelke** [comment]
bors merge

**Pull request closed without merging**